core/src/nats.ts

```diff
@@ -19,22 +19,29 @@ export const connectJetstream = (host: string) => {
 }
 
 export const buildConsumer = async (
-  { connection, stream, options }:
-  { connection: any, stream: string, options: any }
+  { connection, options }:
+  { connection: NatsConnection, options: Partial<ConsumerConfig> }
 ) => {
+  const stream = "DebeziumStream";
   const jetstream = connection.jetstream();
   const jetstreamManager = await connection.jetstreamManager();
   let consumer;
 
   try {
     consumer = await jetstream.consumers.get(stream, options.durable_name);
     const { config } = await consumer.info();
-    if (Object.keys(options).some(key => options[key] !== config[key])) {
+    const hasDifferentValue = Object
+      .keys(options)
+      .some(key => {
+          const keyAsOptionsKeyType = key as keyof typeof options;
+          return options[keyAsOptionsKeyType] !== config[keyAsOptionsKeyType]
+      });
+    if (hasDifferentValue && options.durable_name != null) {
```

Comment:
Nit: what do you think about replacing `&& options.durable_name != null` with `&& options.durable_name`? This will make sure the value is not `undefined`, `null`, or an empty string.